src/jit/lsrabuild.cpp

```diff
@@ -1376,6 +1375,10 @@ LinearScan::buildUpperVectorSaveRefPositions(GenTree* tree, LsraLocation current
 //    currentLoc - The location of the current node
 //    liveLargeVectors - The set of lclVars needing restores (returned by buildUpperVectorSaveRefPositions)
```

Comment:
another question about names, could you please rename it to `liveLargeVectorsVars`?